cmd/nitro/nitro.go

```diff
@@ -759,6 +786,10 @@ func ParseNode(ctx context.Context, args []string) (*NodeConfig, *genericconf.Wa
 	}
 
 	if nodeConfig.Persistent.Chain == "" {
+		if !chainFound {
+			// If persistent-chain not defined, user not creating custom chain
+			return nil, nil, nil, nil, nil, fmt.Errorf("Unknown chain with L1: %d, L2: %d\n", l1ChainId.Uint64(), l2ChainId)
```

Comment:
We might still want to say something about setting the persistent chain since they could have a full custom config?